haystack/indexing/file_converters/docx.py

```diff
@@ -0,0 +1,28 @@
+from haystack.indexing.file_converters.base import BaseConverter
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+class docxToTextConverter(BaseConverter):
+    def __init__(self):
+        try:
+            import docx
+        except ImportError as e:
+            raise Exception(
+                """python-docx is not installed.
+                
+                Please run !pip install python-docx
+                
+                   You can find more details here: https://python-docx.readthedocs.io/en/latest/
+                """
+            )
+    def page_text(self, file_path: Path) -> str:
```

Comment:
Ideally, we have here a similar behavior as for the PDFToTextConverter:
- name of method => extract_pages()
- returning a list of strings (one per page)

It might be that the page breaks are not directly accessible in the docx format. In that case we can return the paragraphs.

Reply:
I've made the following changes

- name of method => extract_pages()
- returning a list of strings (one per paragraph)